### ISSUE TITLE
Error when import data

### ISSUE THREAD

**Issue by myckei** [x4x/pywws2mysql#3]
Hello.
After several years of successful use of pywws2mysql, I encountered a problem today.
Error message when importing station records:
Traceback (most recent call last):
  File "wws2mysql.py", line 278, in <module>
    sys.exit(main())
  File "wws2mysql.py", line 252, in main
    sqldb.insert_in_db( dbcon['table'], all_data[i])
  File "/opt/pywws2mysql/mysql_interface/db_interface.py", line 118, in insert_in_db
    self.insert(table, new_dict)
  File "/opt/pywws2mysql/mysql_interface/db_interface.py", line 108, in insert
    raise NameError("Insert Error! Dataset probably already existing!")
NameError: Insert Error! Dataset probably already existing!

Of course there is a database, there are records in 3 years.
I tried to create a new database, the same error was displayed. I added one record to the database, the same result.

Please help :-)

**Comment by myckei** [comment]
I noticed that there is an error in the values from the weather station and a temperature of -2046.0 C is reported. Could the error be the overlap of the column range in the database?

**Comment by myckei** [comment]
Where does pywws2mysql take data? I deleted records from weatherstation, pywws or easyweather is not retrieving any data. Pywws2mysql displays data until December 25, 2019 and then ends with an error. Mystery :-)

**Comment by x4x** [comment]
Hi,
pywws2mysql uses a module of pywws to read the data out of the wather stations buffer.

**Comment by myckei** [comment]
Hi.

I do not know if there was a problem with unreasonable values in the station or in duplicate records - the same measurement time.
But I deleted the station (using the pywws command, the button on the station screen, and also with easyweather).
But when I ran pywws2mysql, it still retrieved data about a week back.
With the help of a friend, I modified your script in the mysql_interface.py file on line 129 by adding the "pass" command.
After this adjustment, the import will take place and everything seems to work.
I do not know how the records in the station came and I apologize for the interference in the script :-)
